ex_cubic_ingestion/lib/mix/tasks/schedule_dmap.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Mix.Tasks.ScheduleDmap do
+  @moduledoc """
+  This task will insert the ScheduleDmap job upon execution, resulting in
+  the fetching of all DMAP feeds.
+  """
+  use Mix.Task
+
+  alias ExCubicIngestion.Workers.ScheduleDmap
+
+  require Oban
+
+  @doc """
+  Start the application without the GenServers, so we don't have competing
+  processes for querying the incoming bucket, and then inserting a ScheduleDMAP
+  job for Oban to process.
+  """
+  @impl Mix.Task
+  def run(_args) do
+    Application.put_env(:ex_cubic_ingestion, :start_app_children?, false)
+
+    Mix.Task.run("app.start")
+
+    Oban.insert(ScheduleDmap.new(%{}))
```

Comment:
question: what happens if the insert fails?

Reply:
An exception will happen and the job will not get inserted. I'm assuming depending on where it's going to be run (AWS Batch or ECS) the log should indicate that it failed to run.

Reply:
I don't think that's the case: https://hexdocs.pm/oban/Oban.html#insert/2 says that it returns a tuple rather than raising an exception. Did you mean to use [`insert!`](https://hexdocs.pm/oban/Oban.html#insert!/2)?
```suggestion
    Oban.insert!(ScheduleDmap.new(%{}))
```

Reply:
Thanks. Updated.